README.md

```diff
@@ -201,6 +201,22 @@ or if using dockerized Devnet:
 $ docker run -e RUST_LOG=<LEVEL> shardlabs/starknet-devnet-rs
 ```
 
+By default, logging of request and response data is turned off.
+To see the request or/and response body, additional level have to be provided via `RUST_LOG` environment variable.
+To log the request body use `REQUEST`, to log the response body use `RESPONSE`.
+
+NOTE! that logging request and response requires at least logging level `INFO`.
+
+Example:
+
+```
+$ RUST_LOG="REQUEST, RESPONSE" starknet-devnet
+```
+
+```
+$ RUST_LOG="REQUEST, RESPONSE, INFO" starknet-devnet
```

Comment:
I'm getting the following when starting like that:
```
$ RUST_LOG="REQUEST, RESPONSE, INFO" cargo run --release
    Finished release [optimized] target(s) in 0.26s
     Running `target/release/starknet-devnet`
ignoring ` `: invalid filter directive
ignoring ` info`: invalid filter directive
Predeployed FeeToken
ETH Address: 0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7
...
2024-05-03T07:47:12.628819Z  INFO starknet_devnet: Starknet Devnet listening on 127.0.0.1:5050
...
```

Reply:
fixed in [2042e86](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/455/commits/2042e86f9b8207bfec829d94775b4eccd0e5d45d)